api/TextEncoder.json

```diff
@@ -70,86 +82,408 @@
               "version_added": null
             }
           },
           "status": {
-            "experimental": false,
+            "experimental": true,
             "standard_track": true,
             "deprecated": false
           }
         }
       },
-      "encode": {
+      "TextEncoder": {
         "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextEncoder/encode",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextEncoder/TextEncoder",
+          "description": "<code>TextEncoder()</code> constructor",
           "support": {
             "webview_android": {
+              "version_added": "38"
+            },
+            "chrome": [
+              {
+                "version_added": "53",
+                "notes": "Does not accept parameters. Supports only <code>uft-8</code> encoding."
+              },
+              {
+                "version_added": "38",
+                "version_removed": "53",
+                "notes": "Throws <code>RangeError</code> exception for unknown encoding type"
+              }
+            ],
+            "chrome_android": {
+              "version_added": "38"
+            },
+            "edge": {
               "version_added": null
             },
-            "chrome": {
+            "edge_mobile": {
               "version_added": null
             },
-            "chrome_android": {
+            "firefox": [
+              {
+                "version_added": "48",
+                "notes": "Does not accept parameters. Supports only <code>uft-8</code> encoding."
+              },
+              {
+                "version_added": "38",
+                "version_removed": "48",
+                "notes": "Invalid parameter leads to the standard <code>RangeError</code> exception."
+              },
+              {
+                "version_added": "19",
+                "version_removed": "38",
+                "notes": "Invalid parameter leads to a <code>TypeError</code> exception."
+              },
+              {
+                "version_added": "18",
+                "notes": "Slightly different version of the spec"
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "48",
+                "notes": "Does not accept parameters. Supports only <code>uft-8</code> encoding."
+              },
+              {
+                "version_added": "38",
+                "version_removed": "48",
+                "notes": "Invalid parameter leads to the standard <code>RangeError</code> exception."
+              },
+              {
+                "version_added": "19",
+                "version_removed": "38",
+                "notes": "Invalid parameter leads to a <code>TypeError</code> exception."
+              },
+              {
+                "version_added": "18",
+                "notes": "Slightly different version of the spec"
```

Comment:
"Firefox 18 implemented an earlier and slightly different version of the specification."